HW-14/RegEx(14).py

```diff
@@ -0,0 +1,65 @@
+import json
+import re
+data = {
+    'Frodo': '12345678',
+    'Sam': '87654321',
+    'Mari': '18273645',
+
+}
+
+with open('file', 'w+') as file:
+    data = json.dumps(data)
+    file.write(data)
+    print('Open file')
+with open('file', 'r+') as file:
+    file_dict = file.read()
+    file_dict = json.loads(file_dict)
+
+while True:
+    contacts = input('Enter your contact:')
+    if contacts == 'list':
+        print(file_dict)
+    elif contacts == 'stats':
+        print(f'You have {len(file_dict)} contacts.')
+    elif contacts == 'add':
+        new_contacts = input('Enter new contact: ')
+        key = new_contacts
+        with open('file', 'r+') as file:
+            file_dict = file.read()
+            file_dict = json.loads(file_dict)
+        if key in file_dict:
+            print(f'This contact already exist.')
+        else:
+            number = input('Enter number: ')
+            num = int(number)
```

Comment:
You will never have here phone number in the format `+380XXXXXXXXX` since int(number) will remove the `+` sign from the beginning of the string. I don't think you need this transformation here. You can save the number as a string